src/GraphQL/DI/GraphQLBuilderBase.cs

```diff
@@ -18,6 +21,9 @@ public abstract class GraphQLBuilderBase : IGraphQLBuilder
         /// Does not include <see cref="IGraphQLSerializer"/>, and the default <see cref="IDocumentExecuter"/>
         /// implementation does not support subscriptions.
         /// </summary>
+#if NET5_0_OR_GREATER
+        [DynamicDependency(DynamicallyAccessedMemberTypes.PublicConstructors, typeof(ErrorInfoProviderOptions))]
```

Comment:
This line of code should not be needed.  `Configure<T>` has the correct metadata to instruct the AOT trimmer to preserve T (both on the interface and the class within `GraphQL.MicrosoftDI`), and within this method `Configure<ErrorInfoProviderOptions>()` is called.  For whatever reason, it simply doesn't work, and I'm not sure why.

Reply:
>  For whatever reason, it simply doesn't work, and I'm not sure why.

Do you see an runtime exception?

Reply:
Sure.  Something along the lines of "no native code for this constructor found".  Very specifically a trimming error.

Reply:
If I have time, I'll open a few bug reports for these issues, providing I can reliably reproduce them all.  Some will be easy.  Not sure about this one.

Reply:
So keep as is for mow with TODO reminder comment?

Reply:
yes